### ISSUE TITLE
Removing commas from match arms

### ISSUE THREAD

**Issue by nrc** [nrc/rustfmt#355]
```
fn to_opt_strs(self) -> Vec<Option<&'static str>> {
             match self {
-                UnlabelledNodes(len) => vec![None; len],
+                UnlabelledNodes(len) => vec![None; len]
                 AllNodesLabelled(lbls) => lbls.into_iter().map(|l| Some(l)).collect(),
```

Looks like only in the presence of macros of perhaps `]` or something

**Comment by scyptnex** [comment]
caused by Rewrite for Arm interaction with macro-formatting
Fixing this will cause conflicts with the 'match arm collapse' fix
I can add a fix to that branch or just merge it and fix this in isolation

**Comment by nrc** [comment]
I'd prefer a separate PR, I'll review the the match arm collapse PR now

**Comment by nrc** [comment]
Closed by #357
